### DISCUSSION TITLE
RFC(admin-api) OpenAPI3 Schema

### DISCUSSION THREAD

**Discussion by mayocream** [Kong/kong#8289] [Ideas and feature requests]
We might support OpenAPI3 Schema in a link like `/openapiv3/{entity name}` ?
ref: https://docs.konghq.com/gateway/2.7.x/admin-api/#retrieve-entity-schema

**Comment by juzhiyuan** [comment]
Like openapi-to-kong? or kong-to-openapi?

**Reply by mayocream** [reply]
What I concern about is more like Kong schema to OpenAPI spec.

**Reply by juzhiyuan** [reply]
Sure 😉 also cc @membphis
